File: 351001/Strizhak/rest_jpa/rest_jpa/Dtos/UserResponseTo.cs
namespace rest_api.Dtos
{
    public class UserResponseTo
    {
        public long Id { get; set; }
        public string Login { get; set; } = null!;
        public string Firstname { get; set; } = null!;
        public string Lastname { get; set; } = null!;
    }
}

File: 351002/Alexeichik/MyStartRestApplication/publisher/publisher-api/src/main/java/com/github/Lexya06/startrestapp/publisher/api/dto/user/UserResponseTo.java
package com.github.Lexya06.startrestapp.publisher.api.dto.user;

import lombok.*;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserResponseTo {
    Long id;
    String login;
    String password;
    String firstname;
    String lastname;
    String role;
}
